README.md

```diff
@@ -0,0 +1,240 @@
+and `{queue4}`.
+
+An example mapping could look like `email-worker:emails:1;notifications-worker:notifications:2`.
+
+## Configuration Env variables
+
+| Name                        | Description                                                                                                                                                                                                                                                                                     | Default                      |
+|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|
+| KCS_LOG_LEVEL               | The log level to print (panic, fatal, error, warn(ing), info, debug, trace)                                                                                                                                                                                                                     | info                         |
+| KCS_WORKER_STALE_TIME       | When we should mark a worker as lost in seconds.                                                                                                                                                                                                                                                | 10                           |
+| KCS_WORKER_CLEANUP_INTERVAL | How often to clean up lost workers in seconds.                                                                                                                                                                                                                                                  | 5                            |
+| KCS_WORKER_QUEUE_MAP        | When your Celery apps don't send the queue in the heartbeat, you can use this to map the workers by hostname. The format is {identifier1}:{queue1},{queue2}:{concurrency1};{identifier2}:{queue3},{queue4}:{concurrency2}. When the identifier is found in the worker hostname, it will map it. |                              |
+| KCS_REDIS_TYPE              | What type of redis to use, standalone or sentinel.                                                                                                                                                                                                                                              | standalone                   |
+| KCS_REDIS_SERVER            | host:port list, seperated by a comma (standalone uses only the first in case of multiple given)                                                                                                                                                                                                 | localhost:6379               |
```

Comment:
```suggestion
| KCS_REDIS_SERVER            | host:port list, seperated by a comma (redis in standalone uses only the first in case of multiple given)                                                                                                                                                                                                 | localhost:6379               |
```